src/plugins/intel_cpu/src/extension.cpp

```diff
@@ -40,7 +40,10 @@ namespace {
 template <typename Op>
 class TypeRelaxedExtension : public ov::OpExtension<ov::op::TypeRelaxed<Op>> {
```

Comment:
what about the base TypeRelaxed class? do we need to patch it similar way?
do we have something similar on gpu?

Reply:
No the TypeRelaxedBase class do not have static info it only provides common vresion prefix, and some common members as non-template base.
There is no such extension on GPU